samples/nrf9160/modem_shell/src/srest/srest_shell.c

```diff
@@ -0,0 +1,310 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdlib.h>
+#include <zephyr.h>
+#include <assert.h>
+
+#include <shell/shell.h>
+#include <getopt.h>
+
+#include <net/http_parser.h>
+#include <net/rest_client.h>
+
+#include "srest_shell.h"
+#include "mosh_print.h"
+
+static const char srest_shell_cmd_usage_str[] =
+	"Simple REST shell client\n"
+	"Usage:\n"
+	"srest -d host [-p port] [-m method] [-u url] [-b body] [-H header1] [-H header2 ... header10]\n"
+	"               [-s sec_tag] [-v verify_level] [-t timeout_in_ms] [-l length of the response buffer]\n"
```

Comment:
`length_of_the_response_buffer`, or maybe `response_buffer_length` for -l option?

Reply:
Done